Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```diff
@@ -712,6 +747,55 @@ def get_service_keys():
     }
 
 
+def add_responders_to_incident(incident_id, message, user_requests="", escalation_policy_requests="", requestor_id=DEFAULT_REQUESTOR):
```

Comment:
it seems like you are not using the function parameters at all, why is that? what are you trying to achieve here?

Also, what are these definitions ```RequestorID```, ```UserRequests```, ```Message```, ```EscalationPolicyRequests```, ```IncidentID```? 
Those objects are not even defined anywhere.

Reply:
Like I said before, this code is legacy from colleague . I made necessary changes in https://github.com/demisto/content/pull/17745/commits/072e7428c67b5d89c3c1bb6f178b2ac1116c04f4.